backend/seeds/seeds.go

```diff
@@ -72,6 +85,16 @@ func SeedInitialData(store store.Store) (err error) {
 	return initializer.FlagAsInitialized()
 }
 
+// SeedInitialData will seed a store with initial data. This method is
+// idempotent and can be safely run every time the backend starts.
+func SeedInitialData(store store.Store) (err error) {
+	config := Config{
+		AdminUsername: "admin",
+		AdminPassword: "P@ssw0rd!",
```

Comment:
Under what circumstances would we still need to seed `admin`/`P@ssw0rd!` since it's being removed?

Reply:
Two reasons.

1) The integration tests depend on it to some extent.
2) By keeping this function as-is, we avoid breaking the package functionality, and therefore any upstream consumers that depend on it.

Reply:
> The integration tests depend on it to some extent.
Shouldn't the integration tests use the same methods that real users should use?

> By keeping this function as-is, we avoid breaking the package functionality, and therefore any upstream consumers that depend on it.

This is already a breaking change in the first place and would go into the next major version unless I am misunderstanding something so what are we really buying here?

Reply:
From a Go module compatibility point of view, it costs us little-to-nothing to maintain module compatibility here, which is why I've kept this function as-is, while introducing a new one.

Reply:
Also, no, the integration tests should not run `sensu-backend init`. Integration tests are different from functional tests, which we have outside of the sensu-go repository. These functional tests cover running the `sensu-backend init` tool the way a user would.

The integration tests are more concerned with receiving a bootstrapped, working backend at the start of the test context.